combine duplicate doc (#2934)
pingcap/docs: sql-statements/sql-statement-show-table-next-rowid.md

# SHOW TABLE NEXT_ROW_ID

> `SHOW TABLE NEXT_ROW_ID` is used to show the details for some special columns, including

Reviewer: ```suggestion
`SHOW TABLE NEXT_ROW_ID` is used to show the details of some special columns of a table, including:
```

> * `AUTO_INCREMENT` column created automatically, aka `_tidb_rowid` column.

Reviewer: ```suggestion
* `AUTO_INCREMENT` column automatically created by TiDB, namely, `_tidb_rowid` column.
```

## Examples

> For newly created tables, NEXT_GLOBAL_ROW_ID is 1 because no Row ID is allocated.

Reviewer: ```suggestion
For newly created tables, `NEXT_GLOBAL_ROW_ID` is `1` because no Row ID is allocated.
```
